GTG/gtk/plugins.py

```diff
@@ -190,7 +190,8 @@ def _init_plugin_tree(self):
 
         # renderer for the toggle column
         renderer = Gtk.CellRendererToggle()
-        renderer.set_property('xpad', 6)
+        renderer.set_property('margin-start', 6)
```

Comment:
Solve this issue:

```
Traceback (most recent call last):
  File "/home/izidor/gsoc/gtg/GTG/gtk/browser/browser.py", line 430, in open_plugins
    self.vmanager.configure_plugins()
  File "/home/izidor/gsoc/gtg/GTG/gtk/manager.py", line 233, in configure_plugins
    self.plugins.activate()
  File "/home/izidor/gsoc/gtg/GTG/gtk/plugins.py", line 237, in activate
    self._init_plugin_tree()
  File "/home/izidor/gsoc/gtg/GTG/gtk/plugins.py", line 193, in _init_plugin_tree
    renderer.set_property('margin-start', 6)
TypeError: object of type `GtkCellRendererToggle' does not have property `margin-start'
```

(I believe that xpad is not deprecated for GtkCellRenderer: https://developer.gnome.org/gtk3/stable/GtkCellRenderer.html#GtkCellRenderer--xpad )